docs/tutorials/testing_with_control_center.rst

```diff
@@ -33,6 +33,10 @@ section of our website, and individual links are provided below for convenience:
 
 Hardware Setup
 ==============
+.. warning:: **If you are using a power supply to power your module, it is possible  to damage or destroy your module with aggressive commands.** This is because Vertiq modules can also act as generators. 
```

Comment:
I think you should specify what *aggressive commands* are. We know that the biggest problem comes from people stopping hard, but people reading this won't. Even if it is wordier, I think it is best to be as clear as possible about what can cause the issues.